content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -125,31 +63,46 @@ The following example configures the Istio CNI plugin to ignore pods in the name
 | `cniConfDir` | | `/etc/cni/net.d` | Must be the same as the environment's `--cni-conf-dir` setting (`kubelet` parameter). |
 | `cniConfFileName` | | None | Leave unset to auto-find the first file in the `cni-conf-dir` (as `kubelet` does).  Primarily used for testing `install-cni` plugin configuration.  If set, `install-cni` will inject the plugin configuration into this file in the `cni-conf-dir`. |
 
-### Hosted Kubernetes Usage
+### Excluding specific Kubernetes namespaces
+
+This example configures the Istio CNI plugin via `helm template` to modify the log level and
```

Comment:
Readers can be confused if they don't know that `helm template` is just a Helm command. User lists instead of paragraphs to make the content more scannable. This applies to all lists in paragraphs but I am only leaving a suggestion here:

```suggestion
This example uses Helm to perform the following tasks:

- Configure the Istio CNI plugin 
- Modify the log level
- Ignore the pods in the following namespaces:
    - `istio-system`
    - `foo_ns`
    - `bar_ns`
```

Reply:
Done.